TreeTracker/UI/Views/Profile Screen/ProfileViewController.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  ProfileViewController.swift
+//  TreeTracker
+//
+//  Created by Remi Varghese on 3/23/21.
+//  Copyright © 2021 Greenstand. All rights reserved.
+//
+
+import UIKit
+
+class ProfileViewController: UIViewController {
+    var trees: [Tree] = []
+    weak var rootcoordinator: RootCoordinator!
+    var homecoordinator: HomeCoordinator!
+    // MARK: - IBOutlets
+    @IBOutlet weak var profilepic: UIImageView!
```

Comment:
Should we make this var private so it can't be access from outside the view?

Reply:
Also the variable name - we should try to avoid abreviations and also be consistent with the format of the other vriable names. Maybe something like `profileImageView` or `profilePictureImageView`would be better?

Reply:
> Should we make this var private so it can't be access from outside the view?

is that about rootcoordinator and homecoordinator

Reply:
I declared the properties rootcoordinator and homecoordinator as private. The name of IBoutlet for image is also changed

Reply:
I made the IBOutlet as private